src/main/resources/application.yml

```diff
@@ -78,6 +78,7 @@ nakadi:
   auth:
     plugin:
       factory: org.zalando.nakadi.plugin.auth.DefaultApplicationServiceFactory
+  event.max.bytes: 1000000
```

Comment:
@adyach Why you expect that if you know that Nakadi validates event size as well?